ddtrace/tracer/textmap.go

```diff
@@ -896,6 +898,11 @@ func parseTraceparent(ctx *spanContext, header string) error {
 // `origin` = `o`
 // `_dd.p.` prefix = `t.`
 func parseTracestate(ctx *spanContext, header string) error {
+	if header == "" {
```

Comment:
Do we need to do this empty check in both places? (If so maybe we should duplicate this comment in both places to make it obvious why it's alright for it to be empty)

Reply:
Good point, the empty check here should be enough! (PR updated)